packages/nx/src/utils/find-matching-projects.ts

```diff
@@ -0,0 +1,35 @@
+import minimatch = require('minimatch');
+
+export function findMatchingProjects(
```

Comment:
since you extracted this into a separate utility which is used in more than place, you can add some tests for this utility

Reply:
Yeah, thats a good shout out. I'll get them added.